src/transformers.jl

```diff
@@ -106,8 +114,9 @@ Optionally specify the `dims` to apply the [`Transform`](@ref) along certain dim
 function apply!(A::AbstractArray, t::Transform; dims=:, kwargs...)
     dims == Colon() && return _apply!(A, t; kwargs...)
 
-    for x in eachslice(A; dims=dims)
-        _apply!(x, t; kwargs...)
+    _dims = invert_dims(A, dims)  # opposite convention to `eachslice`
```

Comment:
Can you add a blurb on what direction dims and inds correspond to to this docstring?